assets/js/blocks/price-filter/edit.js

```diff
@@ -151,24 +152,20 @@ export default function( { attributes, setAttributes } ) {
 		</Placeholder>
 	);
 
-	const TagName = `h${ headingLevel }`;
-
 	return (
 		<Fragment>
 			{ PRODUCT_COUNT === 0 ? (
 				noProductsPlaceholder()
 			) : (
 				<div className={ className }>
 					{ getInspectorControls() }
-					<TagName>
-						<PlainText
-							className="wc-block-attribute-filter-heading"
```

Comment:
It looks like the class was wrong here too (it should be `wc-block-price-filter-heading`).